### ISSUE TITLE
Sticky Scroll not scaling with editor text size  (live example in attachment)

### ISSUE THREAD

**Issue by nikhil-swamix** [microsoft/vscode#195859]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.83.1
- OS Version: win10

Steps to Reproduce:
1. press control and zoom out (using mouse scroll wheel)
2. we don't see the sticky scroll with equal font size of editor text, (which stay static for a given indent level)

![image](https://github.com/microsoft/vscode/assets/54004431/c08e8750-8617-4ae8-9e71-9b7996a2f126)

**Comment by ADTC** [comment]
Can you try applying this in your `settings.json` to alter the background color and see what's going on?

```jsonc
  "workbench.colorCustomizations": {
    "editorStickyScroll.background": "#0f0" // green
  },
```

What does Ctrl + scroll wheel do? Reduce the font size of editor only? I have tried both "View: Zoom Out" and "Editor Font Zoom Out" from the command palette, but can't reproduce. But I'm still on `1.76.0` coz I don't update often.

I think I might have a related issue:

Broken sticky scroll in SCSS diff:

<img width="458" alt="image" src="https://github.com/microsoft/vscode/assets/6047296/596924ec-2c7c-4c05-bc59-035f69719d08">

Incomplete sticky scroll in TSX diff:

<img width="441" alt="image" src="https://github.com/microsoft/vscode/assets/6047296/f5ee2366-6749-4592-ae60-60d50891e021">

Maybe I have to update to the latest version to check if it's already fixed 😄

(Note that I am using "Input Sans" as my font.)

**Comment by glebbash** [comment]
Still not fixed (1.85.0). This is really annoying 😤
![image](https://github.com/microsoft/vscode/assets/17786588/d2047be4-7087-429c-acab-6e9c7e6a902c)
.

**Comment by aiday-mar** [comment]
I believe this should be fixed now
